pupil-spa/src/app/submission-failed/submission-failed.component.html

```diff
@@ -6,5 +6,12 @@ <h1 class="heading-xlarge">Unable to connect</h1>
     <div class="panel panel-border-wide">
       Please contact the helpline on {{supportNumber}} for advice on how to upload the check.
     </div>
+    <form (ngSubmit)="onSubmit()" autocomplete="off" autocapitalize="none">
```

Comment:
autocomplete and autocapitalise aren't really doing anything here.  Why does this need a form rather than a `[routerLink]`?

Reply:
@jon-shipley updated